inference_segmentation.py

```diff
@@ -401,6 +403,11 @@ def main(params: dict) -> None:
         logging.warning(f'\nDebug mode activated. Some debug features may mobilize extra disk space and '
                         f'cause delays in execution.')
 
+    # list of GPU devices that are available and unused. If no GPUs, returns empty dict
+    gpu_devices_dict = get_device_ids(num_devices, max_used_ram_perc=max_used_ram, max_used_perc=max_used_perc)
+    chunk_size = get_key_def('chunk_size', params['inference'], default=512, expected_type=int)
+    chunk_size = calc_inference_chunk_size(gpu_devices_dict=gpu_devices_dict, max_pix_per_mb_gpu=50, default=chunk_size)
```

Comment:
just a note: calculating chunk size dynamically here specifically at inference may cause problems with test time augmentation and smoothing. A default size of 512 is advised. 512 is padded to 512 * 2 during test time routines

Reply:
ok, do you agree to keep the "automatic chunk_size" option? it's meant to be tunable, i.e. max_pix_per_mb_gpu can be lowered if the automatic way busts memory.

Reply:
As discussed we can lower the  `max_pix_per_mb_gpu` for now, keeping in mind that we could later optimize its definition during inference.

Reply:
True. I'll address this in PR #276